FISH-1353 Payara VSCode extension 1.1.0 features (#177)
payara/Payara-Community-Documentation: docs/modules/ROOT/pages/documentation/ecosystem/vscode-extension/payara-server.adoc

image::vscode-extension/payara-server/deploy-application.png[alt=Deploy application, width="60%"]

> Alternatively application can be deployed from *Java Projects* interface:

Reviewer: ```suggestion
Alternatively, an application can be deployed from *Java Projects* interface:
```

image::vscode-extension/payara-server/java-projects-deploy.png[alt=Deploy application, width="60%"]

> The deploy application can be opened in the browser, undeployed or disabled by right-clicking on the application:

Reviewer: ```suggestion
The deployed application can be opened in the browser, undeployed or disabled by right-clicking on the application:
```

image::vscode-extension/payara-server/list-application.png[alt=Open application homepage, width="60%"]

[[deployment-settings]]
== Deployment Settings

The Auto Deploy and Hot Deploy mode are helpful for developers to run and test 
an application immediately after making changes to its sources without 
restarting the Server or manual redeployment to maximize your productivity.

[[hot-deploy]]
=== Hot Deploy

> _Since *5.201*&nbsp;_ Payara Server Community provides the Hot Deployment capability by boosting the performance 

Reviewer: ```suggestion
_Since *5.201*&nbsp;_

Payara Server provides the Hot Deployment capability, which boosts the performance
```

The service team suggests that we don't mention the edition name so that it can be easily copied to both editions.

of the application redeployment operation and dynamically replaces the classloader of the application on each redeployment in the hot deploy mode.

> The Hot Deploy mode can be enabled in the Visual Studio Code from the *Deployment Settings* of the Payara Server.

Reviewer: ```suggestion
The Hot Deploy mode can be enabled in Visual Studio Code from the *Deployment Settings* of Payara Server.
```

image::vscode-extension/payara-server/deployment-settings.png[Deployment Settings]
image::vscode-extension/payara-server/deployment-settings-options.png[Enable Hot Deploy]

NOTE: The Hot Deploy feature is in technical preview.

> After enabling the Hot Deploy settings as mentioned above, Simply deploy the Web Applications to Payara Server Community

Reviewer: ```suggestion
After enabling the Hot Deploy settings as mentioned above, simply deploy web applications to Payara Server 
```

and modify any Java class of the application to perform the auto redeploy of application on save 

> and observer the fast redeployment of the application in Payara Server console log in compare to vanilla redeployment.

Reviewer: ```suggestion
. You'll observe that redeployment of the application in Payara Server is faster than vanilla redeployment and console log will confirm that the hot redeployment happened.
```

[[auto-deploy]]
=== Auto Deploy
By enabling the Auto Deploy feature, you may automate the manual deployment 
processes to boost the developers productivity. It will undeploy the existing 
application and redeploy the new application on every save action so developers 

> can focus on the key development task.

Reviewer: I would mention that this feature full redeploys an application and therefore is slower than hot redpeloy, which redeploys only the changed part of an application.
